sycl/doc/GettingStartedAlveo.md

```diff
@@ -0,0 +1,449 @@
+..............................
+INFO: Loading new firmware on SC
+.
+Updating shell on card[0]
+INFO: ***Found 701 ELA Records
+Idcode byte[0] ff
+Idcode byte[1] 20
+Idcode byte[2] bb
+Idcode byte[3] 21
+Idcode byte[4] 10
+Enabled bitstream guard. Bitstream will not be loaded until flashing is finished.
+Erasing flash...................................
+Programming flash...................................
+Cleared bitstream guard. Bitstream now active.
+1 Card(s) flashed successfully.
+Cold reboot machine to load the new image on FPGA.
+```
+
+Unfortunately you need to reboot the machine to have the new shell
+loaded inside the FPGA.
+
+Then you can check with a pre-compiled FPGA program provided by the
+shell that the board is working correctly with:
+```bash
+sudo /opt/xilinx/xrt/bin/xbutil validate
+INFO: Found 1 cards
+
+INFO: Validating card[0]: xilinx_u200_xdma_201830_1
+INFO: Checking PCIE link status: PASSED
+INFO: Starting verify kernel test: 
+INFO: verify kernel test PASSED
+INFO: Starting DMA test
+Host -> PCIe -> FPGA write bandwidth = 11127.5 MB/s
+Host <- PCIe <- FPGA read bandwidth = 12147.9 MB/s
+INFO: DMA test PASSED
+INFO: Starting DDR bandwidth test: ..........
+Maximum throughput: 47665.777344 MB/s
+INFO: DDR bandwidth test PASSED
+INFO: Starting P2P test
+P2P BAR is not enabled. Skipping validation
+INFO: P2P test PASSED
+INFO: Starting M2M test
+bank0 -> bank1 M2M bandwidth: 12089.7 MB/s	
+bank0 -> bank2 M2M bandwidth: 12064.7 MB/s	
+bank0 -> bank3 M2M bandwidth: 12084 MB/s	
+bank1 -> bank2 M2M bandwidth: 12064.1 MB/s	
+bank1 -> bank3 M2M bandwidth: 12066.9 MB/s	
+bank2 -> bank3 M2M bandwidth: 12125.8 MB/s	
+INFO: M2M test PASSED
+INFO: Card[0] validated successfully.
+
+INFO: All cards validated successfully.
+```
+
+It looks there is a problem in the Linux `udev` configuration not
+making the DRM driver accessible for some `other` users. A crude
+work-around for now is for example to run
+```bash
+sudo chmod 666 /dev/dri/renderD129
+```
+if `/dev/dri/renderD129` corresponds to the FPGA board (it might be a
+number different from `129`) and check that it is working by running
+```bash
+clinfo
+```
+that should expose some OpenCL device parameters for the FPGA board.
+
+
+## Compile the SYCL compiler
+
+```bash
+# Get the source code
+git clone --branch sycl/unified/master git@github.com:triSYCL/sycl.git
+cd sycl
+SYCL_HOME=`pwd`
+OPENCL_HEADERS=/usr/include
+export XILINX_XRT=/opt/xilinx/xrt
+mkdir $SYCL_HOME/build
+cd $SYCL_HOME/build
+cmake -DCMAKE_BUILD_TYPE=Release \
+  -DLLVM_CXX_STD="c++17" \
+  -DCMAKE_EXPORT_COMPILE_COMMANDS=1 \
+  -DOpenCL_INCLUDE_DIR=$OPENCL_HEADERS \
+  -DLLVM_ENABLE_PROJECTS="clang" \
+  -DLLVM_EXTERNAL_SYCL_SOURCE_DIR=$SYCL_HOME/sycl \
+  -DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=$SYCL_HOME/llvm-spirv \
+  -DLLVM_TOOL_SYCL_BUILD=ON -DLLVM_TOOL_LLVM_SPIRV_BUILD=ON $SYCL_HOME/llvm
+make -j`nproc` check-all
+```
+
+Some checks are still failing but that does not seem to be an issue.
```

Comment:
"Some checks are still failing but that does not seem to be an issue." -> "Some checks may fail but that may not be an issue" perhaps we should future proof the sentence?